src/main/java/org/gridsuite/explore/server/services/ExploreService.java

```diff
@@ -204,6 +204,17 @@ public void deleteElement(UUID id, String userId) {
         }
     }
 
+    public void deleteElements(List<UUID> uuids, String userId) {
+        try {
+            uuids.forEach(id -> directoryService.deleteElement(id, userId));
+            directoryService.deleteDirectoryElements(uuids, userId);
+            // FIXME dirty fix to ignore errors and still delete the elements in the directory-server. To delete when handled properly.
+        } catch (Exception e) {
+            LOGGER.error(e.toString(), e);
+            directoryService.deleteDirectoryElements(uuids, userId);
```

Comment:
Why this method is called in both try and catch !?

Reply:
I used the same logic of this PR: https://github.com/gridsuite/explore-server/pull/27. we ignore the exception and do the deletion anyway

Reply:
consider this suggestion 
public void deleteElements(List<UUID> uuids, String userId) {
        try {
            uuids.forEach(id -> directoryService.deleteElement(id, userId));
            // FIXME dirty fix to ignore errors and still delete the elements in the directory-server. To delete when handled properly.
        } catch (Exception e) {
            LOGGER.error(e.toString(), e);
        } finally {
            directoryService.deleteDirectoryElements(uuids, userId);
        }
    }

Reply:
done